docs/resource-schemas.md

```diff
@@ -123,17 +134,18 @@ library:
+| `id`              | Every library needs an identifier unique to the platform. Contact the [Carbon Platform Devs](https://github.com/orgs/carbon-design-system/teams/carbon-platform-devs) to receive an `id` when registering a new library. See [identifiers](#identifiers) for more info. | Required | String  | No          | –                         | –            |
+| `name`            | Library display name. Use title-case capitalization.                                                                                                                                                                                                                    | Required | String  | No          | –                         | –            |
+| `description`     | Library description ideally between 50-160 characters in length. Use sentence-case capitalization. Defaults to the `package.json` description if not set here.                                                                                                          | Optional | String  | No          | Value from `package.json` | –            |
+| `inherits`        | Inherit properties from another library on a per-asset basis. See [library inheritance](#library-inheritance).                                                                                                                                                          | Optional | String  | No          | –                         | –            |
+| `packageJsonPath` | Relative location of the library's `package.json`. This is used to reference the library's license, version, code package, and other information.                                                                                                                       | Optional | String  | No          | `/package.json`           | –            |
+| `externalDocsUrl` | Absolute URL to externally-hosted documentation.                                                                                                                                                                                                                        | Optional | String  | No          | –                         | –            |
+| `demoLinks`       | Links to demo sites. See [demo links](#demo-links).                                                                                                                                                                                                                     | Optional | Array   | No          | –                         | –            |
+| `designKits`      | Design kits that contain the design counterpart to assets in this library. See [design kit schema](#design-kit-schema) for more info.                                                                                                                                   | Optional | Object  | Yes         | –                         | –            |
```

Comment:
should we add a designKits key to the library schema example above? ^^

Reply:
Done.